.github/workflows/ci.yml

```diff
@@ -63,7 +68,7 @@ jobs:
         run: |
           sudo apt-get update
           sudo apt-get install libkrb5-dev
-          pip install .[tests]
+          pip install .[tests] sqlalchemy~=${{ matrix.sqlalchemy }}
```

Comment:
I'd instead say to inline the constraint in the matrix entry itself. 

i.e. `~=1.4.0` (latest 1.4)
`==1.4.2` (some specific version)

Reply:
Makes sense!